shoebot/grammar/nodebox.py

```diff
@@ -350,7 +350,7 @@ def drawimage(self, image, x=None, y=None):
             x = image.x
         if y is None:
             y = image.y
-        self.image(image.path, image.x, image.y, data=image.data)
+        return self.image(image.path, x, y, data=image.data)
```

Comment:
Did this to make the test easier to write, but shouldn't harm anything ?

Reply:
yep, i think this is proper usage of Grob objects -- saving them into a variable doesn't affect their drawing (or not) into the canvas

Reply:
I'm not convinced I should save to a variable.. you get a new image that is the one that was drawn, but it's kind of a weird implementation detail leaking to the surface.

I should probably use a mock or something to find out what coordinates where used to draw it to.